src/org/labkey/test/components/glassLibrary/grids/DetailTableEdit.java

```diff
@@ -252,24 +290,103 @@ public DetailTableEdit clearSelectValue(String fieldCaption)
         return this;
     }
 
-    protected static abstract class Locators
+    private String getSourceTitle()
```

Comment:
This can be reduced to just:
`return getTitle().replace("editing ", "");`

Reply:
consider it done